fault/system_verilog_target.py

```diff
@@ -860,7 +868,6 @@ def ncsim_cmd(self, sources, cmd_file):
         cmd += self.def_args(prefix='+define+')
 
         # misc flags
-        cmd += ['-access', '+rwc']
```

Comment:
Is this okay to turn off in general? I don't remember why this option is included.  @sgherbst do you use this option ever? I think this may have been copied over from an old ncsim script that Nikhil gave me as an example, so I'm not sure if it's actually used by anyone.

Based on this https://www.edaboard.com/showthread.php?23842-What-is-the-access-rwc-option-in-NCSIM it seems that this enables access to objects in the design.  We might need this for working with internal signals? (But the tests still pass so that may not be the case)

Reply:
I think “access -r” is needed if you want to dump waveforms, so we would want to enable at least that if the user has requested waveform probing (I sort of remember trying without in the past and getting an empty waveform database).  However, this option shouldn’t be added when not needed because it slows down simulation.  

“access -rw” further enables write access to signals from TCL, and “access -rwc” add connectivity access for driver tracing for debugging.  These options can be useful, but further slow down simulation.

Reply:
`-rwc` is needed if you're using VPI or TCL related simulation where the runtime library/scripts needs access to the simulation objects, especially the ability to override values. This is unrelated to the signal access in test benches since the access is known during compilation. As Steven said, this will slow down simulation simulation.

I think `-access r` is required to dump the VCD files, which I forgot to check in this PR.